tests/api/test_api.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import patch
+
+import responses
+import schemathesis
+
+from tests.fixtures.mock_factory import get_mocked_geolocation_factory
+
+__FACTORY = get_mocked_geolocation_factory()
+with patch('app.geolocation.factory.Factory.get_instance', return_value=__FACTORY):
+    from app.main import app
+
+schema = schemathesis.from_path("openapi/openapi.yml", app=app)
+
+
+@responses.activate
+@schema.parametrize()
+def test_api(case):
+    url = 'https://e-10250.adzerk.net/udb/10250/'
+    responses.add(responses.DELETE, url, status=200)
+
+    responses.delete(url, status=200)
```

Comment:
these lines add mock responses for requests to `url`, right? I'm surprised that we only have to have something for the delete endpoint.

how does this work for the spoc fetching case, wouldn't that endpoint also want to make a different call to kevel?

Reply:
Yeah, that is a good question =) Let me dig into this a bit; I wonder if those requests are just going through because they don't use `requests`

Reply:
Welp, yes that's exactly what's happening. Great catch, thanks Daniel! I'll mock those requests too.